tests/test_config_space_from_yaml.py

```diff
@@ -0,0 +1,362 @@
+from copy import deepcopy
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Any, Dict
+
+import pytest
+import yaml
+
+from embeddings.hyperparameter_search.configspace import BaseConfigSpace
+from embeddings.hyperparameter_search.flair_configspace import FlairModelTrainerConfigSpace
+from embeddings.hyperparameter_search.flair_configspace import (
+    SequenceLabelingConfigSpace as FlairSequenceLabelingConfigSpace,
+)
+from embeddings.hyperparameter_search.lighting_configspace import (
+    LightingTextClassificationConfigSpace,
+)
+
+
+@pytest.fixture(scope="module")
+def output_path() -> "TemporaryDirectory[str]":
+    return TemporaryDirectory()
```

Comment:
replace this fixture with this https://github.com/CLARIN-PL/embeddings/blob/f7c7ee810a9b3fc907edd00631e36a453b962526/tests/hps/conftest.py#L7

it is a more convenient way to use temp paths in pytest. https://docs.pytest.org/en/6.2.x/tmpdir.html

Reply:
fixed